README.md

```diff
@@ -79,6 +79,7 @@ uom = {
         "bigint", "biguint", # Arbitrary width integer storage types.
         "rational", "rational32", "rational64", "bigrational", # Integer ratio storage types.
         "f32", "f64", # Floating point storage types.
+	"serde", # Serde support.
```

Comment:
Looks like this line didn't get indented.

Reply:
oops, that was a sneaky tab character.